src/app/ngrx/effects/cms.effects.ts

```diff
@@ -1,12 +1,9 @@
 import { Injectable } from '@angular/core';
 import { Observable } from 'rxjs/Observable';
+import { catchError, first, map, mergeMap, switchMap } from 'rxjs/operators';
 import 'rxjs/add/observable/of';
```

Comment:
Do these still need to be `/add/`-ed then?  Or just too much churn to change all the `Observable.of`s?

Reply:
`of` is also being added in the styleguide, so we're good on it as is, but could move to using it like https://www.learnrxjs.io/operators/creation/of.html instead of off Observable. eh